### PR TITLE
Fix bridge benchmarking

### PR DESCRIPTION
**Pull request by rjman-ljm**
Close:
- #992

### CODE REVIEW

File: pallets/bridge/src/weights.rs [hunk 1]
```diff
@@ -46,102 +46,27 @@ use frame_support::{
...
-            .saturating_add(T::DbWeight::get().writes(2 as Weight))
-    }
-    fn materialize() -> Weight {
-        (136_015_000 as Weight)
-            .saturating_add(T::DbWeight::get().reads(7 as Weight))
-            .saturating_add(T::DbWeight::get().writes(2 as Weight))
-    }
-}
+impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {	fn set_vote_threshold() -> Weight {
+		(13_000_000 as Weight)			.saturating_add(T::DbWeight::get().reads(1 as Weight))			.saturating_add(T::DbWeight::get().writes(1 as Weight))	}	fn register_chain() -> Weight {
```
Comment (GopherJ) [inline pallets/bridge/src/weights.rs:54]: should be formatted

File: pallets/bridge/src/weights.rs [hunk 2]
```diff
@@ -46,102 +46,27 @@ use frame_support::{
...
-            .saturating_add(RocksDbWeight::get().reads(7 as Weight))
-            .saturating_add(RocksDbWeight::get().writes(2 as Weight))
-    }
-}
+impl WeightInfo for () {	fn set_vote_threshold() -> Weight {
+		(13_000_000 as Weight)			.saturating_add(RocksDbWeight::get().reads(1 as Weight))			.saturating_add(RocksDbWeight::get().writes(1 as Weight))	}	fn register_chain() -> Weight {
+		(14_000_000 as Weight)			.saturating_add(RocksDbWeight::get().reads(1 as Weight))			.saturating_add(RocksDbWeight::get().writes(1 as Weight))	}	fn unregister_chain() -> Weight {
+		(14_000_000 as Weight)			.saturating_add(RocksDbWeight::get().reads(1 as Weight))			.saturating_add(RocksDbWeight::get().writes(1 as Weight))	}	fn register_bridge_token() -> Weight {
+		(17_000_000 as Weight)			.saturating_add(RocksDbWeight::get().reads(2 as Weight))			.saturating_add(RocksDbWeight::get().writes(2 as Weight))	}	fn unregister_bridge_token() -> Weight {
+		(16_000_000 as Weight)			.saturating_add(RocksDbWeight::get().reads(1 as Weight))			.saturating_add(RocksDbWeight::get().writes(2 as Weight))	}	fn set_bridge_token_fee() -> Weight {
```
Comment (GopherJ) [inline pallets/bridge/src/weights.rs:69]: should be formatted

**Comment by GopherJ** [comment]
TODO:

- [ ] Missing `makefile` target
- [ ] ensure the runtime is using correct weights file
- [ ] can you add also `bench-xcm-helper` make target, thanks

Otherwise LGTM

**Comment by rjman-ljm** [comment]
@GopherJ 
Sure

File: pallets/bridge/src/weights.rs (hunk 1 above)
Comment (rjman-ljm) [reply pallets/bridge/src/weights.rs]: Something strange happens, these are formatted and make lint was passed 🤣

**Pull request merged**